build.sbt

```diff
@@ -54,6 +54,7 @@ val elasticsearch5Version = "5.6.16"
 val elasticsearch6Version = "6.8.7"
 val elasticsearch7Version = "7.6.1"
 val featranVersion = "0.5.0"
+val flinkVersion = "1.9.2"
```

Comment:
Why not 1.10 ?

Reply:
I'm actually keeping tabs with beam since the runners are `flink` "specific"

Reply:
FYI Flink 1.10 will be supported in upcoming Beam releases (starting with Beam 2.21 IIRC).

Since Flink jobs are not backwards-compatible with older clusters, I suggest supporting multiple Flink versions going forward if possible, as Beam does (Beam usually tries to support the latest 3 Flink releases).

https://github.com/apache/beam/tree/master/runners/flink